### PR TITLE
feat: a second view function to display the content of an html formha…

### PR DESCRIPTION
**Pull request by Fotsingboris**
I created a new file forms.html and i used it to configure a second function to display the content of an html file. Then i configured the new file in the views.py, models.py and urls.py

### CODE REVIEW

File: devtoy/urls.py [hunk 1]
```diff
@@ -15,8 +15,10 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from django.views import *
+
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
+    path('', forms, name="forms"),
```
Comment (yokwejuste) [inline devtoy/urls.py:22]: This is wrong and doesn't corresponds to the way it should be done.

@Fotsingboris and @AtudJerry  try the following instead.
- Create a **urls.py** for your app you created.
- Reference the new urls.py file `toy/urls.py` in the projects urls.py `devtoy/urls..py`.
- Then use instead the new urls.py to reference all your app's paths to make it follow better the object oriented programming model.

File: toy/models.py [hunk 2]
```diff
@@ -1,3 +1,10 @@
 from django.db import models
 
 # Create your models here.
+class Forms(models.Model):
+    album_name = models.CharField(max_length=35, null=False, unique=True)
+    artist_name = models.CharField(max_length=35, null=False, unique=True)
+    album_picture = models.ImageField(upload_to="media/")
```
Comment (yokwejuste) [inline toy/models.py:7]: The names of the Form fields are bad😅, we're talking here of a toy registry app, what is `artist_name` or  `album_picture` doing there?

@Fotsingboris change them in a new PR and take into consideration all my comments.

File: toy/views.py [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
 # Create your views here.
 def home(request):
-    return render(request,'index.html')
\ No newline at end of file
+    return render(request,'index.html')
+
+def forms(request):
+    if request.method == "POST":
+        album_name = request.POST.get("album_name")
+        artist_name = request.POST.get("artist_name")
```
Comment (yokwejuste) [inline toy/views.py:12]: All these names are not proper guys, Please work them out and please 🙏 discuss and interaction, the discussion is mehn for that.

**Review by yokwejuste — COMMENTED** [review]
@Fotsingboris and @AtudJerry please read through well. We'll shortly discuss on all this.

File: toy/views.py (hunk 3 above)
Comment (Fotsingboris) [reply toy/views.py:12]: please stive I have a problem, I did all the changes you said but when I pushed it my branch is 2 commits ahead and one commit behind of wise-view master and also I can not create a pull request

**Pull request merged**